src/main/java/makamys/neodymium/renderer/NeoRenderer.java

```diff
@@ -432,56 +432,6 @@ private void updateUniforms(double alpha, int pass) {
         fogStartEnd.position(0);
     }
 
-    private void initAttributesRPLEAndShaders() {
-        attributes.addAttribute("POS", 3, 4, GL_FLOAT);
-        attributes.addAttribute("TEXTURE", 2, 4, GL_FLOAT);
-        attributes.addAttribute("COLOR", 4, 1, GL_UNSIGNED_BYTE);
-        attributes.addAttribute("BRIGHTNESS_RED", 2, 2, GL_SHORT);
-        attributes.addAttribute("ENTITY_DATA_1", 1, 4, GL_UNSIGNED_INT);
-        attributes.addAttribute("ENTITY_DATA_2", 1, 4, GL_UNSIGNED_INT);
-        attributes.addAttribute("NORMAL", 3, 4, GL_FLOAT);
-        attributes.addAttribute("TANGENT", 4, 4, GL_FLOAT);
-        attributes.addAttribute("MIDTEXTURE", 2, 4, GL_FLOAT);
-        attributes.addAttribute("BRIGHTNESS_GREEN", 2, 2, GL_SHORT);
-        attributes.addAttribute("BRIGHTNESS_BLUE", 2, 2, GL_SHORT);
-        attributes.addAttribute("EDGE_TEX", 2, 4, GL_FLOAT);
-    }
-
-    private void initAttributesShaders() {
-        attributes.addAttribute("POS", 3, 4, GL_FLOAT);
-        attributes.addAttribute("TEXTURE", 2, 4, GL_FLOAT);
-        attributes.addAttribute("COLOR", 4, 1, GL_UNSIGNED_BYTE);
-        attributes.addAttribute("BRIGHTNESS", 2, 2, GL_SHORT);
-        attributes.addAttribute("ENTITY_DATA_1", 1, 4, GL_UNSIGNED_INT);
-        attributes.addAttribute("ENTITY_DATA_2", 1, 4, GL_UNSIGNED_INT);
-        attributes.addAttribute("NORMAL", 3, 4, GL_FLOAT);
-        attributes.addAttribute("TANGENT", 4, 4, GL_FLOAT);
-        attributes.addAttribute("MIDTEXTURE", 2, 4, GL_FLOAT);
-    }
-
-    private void initAttributesRPLE() {
-        attributes.addAttribute("POS", 3, 4, GL_FLOAT);
-        if (Config.shortUV) {
-            attributes.addAttribute("TEXTURE", 2, 2, GL_UNSIGNED_SHORT);
-        } else {
-            attributes.addAttribute("TEXTURE", 2, 4, GL_FLOAT);
-        }
-        attributes.addAttribute("COLOR", 4, 1, GL_UNSIGNED_BYTE);
-        attributes.addAttribute("BRIGHTNESS_RED", 2, 2, GL_SHORT);
-        attributes.addAttribute("BRIGHTNESS_GREEN", 2, 2, GL_SHORT);
-        attributes.addAttribute("BRIGHTNESS_BLUE", 2, 2, GL_SHORT);
-    }
-
-    private void initAttributesVanilla() {
-        attributes.addAttribute("POS", 3, 4, GL_FLOAT);
-        if (Config.shortUV) {
-            attributes.addAttribute("TEXTURE", 2, 2, GL_UNSIGNED_SHORT);
-        } else {
-            attributes.addAttribute("TEXTURE", 2, 4, GL_FLOAT);
-        }
-        attributes.addAttribute("COLOR", 4, 1, GL_UNSIGNED_BYTE);
-        attributes.addAttribute("BRIGHTNESS", 2, 2, GL_SHORT);
-    }
     /**
      * @implSpec The attributes here need to be kept in sync with {@link MeshQuad#writeToBuffer(BufferWriter, int)}
```

Comment:
This should be `RenderUtil#writeMeshQuadToBuffer` now. (I'll just fix that myself after the merge.)